library/ecdsa.c

```diff
@@ -37,11 +37,9 @@
 #include "mbedtls/asn1write.h"
 
 #include <string.h>
-
 #if defined(MBEDTLS_ECDSA_DETERMINISTIC)
 #include "mbedtls/hmac_drbg.h"
 #endif
-
```

Comment:
I think this change and the one in line 40 are only style so not directly related to this PR...